pxr/usd/usdGeom/mesh.h

```diff
@@ -83,22 +83,22 @@ class SdfAssetPath;
 /// (https://graphics.pixar.com/opensubdiv/docs/subdivision_surfaces.html).
 /// 
 /// \anchor UsdGeom_Mesh_Primvars
-/// <b>A Note About Primvars</b>
+/// __A Note About Primvars__
 /// 
 /// The following list clarifies the number of elements for and the
 /// interpolation behavior of the different primvar interpolation types
 /// for meshes:
 /// 
-/// \li <b>constant</b> One element for the entire mesh; no interpolation.
-/// \li <b>uniform</b> One element for each face of the mesh; elements are
+/// - __constant__: One element for the entire mesh; no interpolation.
```

Comment:
Lovely - I forgot to answer your list-formatting question, but this is the preferred form!